.ci/e2e-synthetics.groovy

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env groovy
```

Comment:
Quite basic pipeline to only checkout the source code and notify the status in slack/email.


TBC in a follow up